src/content/docs/dropins/payment-services/containers/apple-pay.mdx

```diff
@@ -16,28 +16,58 @@ The `ApplePay` container provides the following configuration options:
   compact
   options={[
     ['Option', 'Type', 'Req?', 'Description'],
-    ['getCartId', 'function', 'Yes', 'Should return a promise that resolves to the shopper`s cart ID.'],
-    ['isVirtualCart', 'boolean', 'Yes', 'Whether the cart contains only virtual/downloadable products.'],
+    ['location', 'string', 'Yes', 'Location where the Apple Pay button is to be rendered. Must be either "CHECKOUT" or "PRODUCT_DETAIL".'],
+    ['getCartId', 'function', 'Maybe', 'Required if createCart not provided. Should return a promise that resolves to the shopper\'s cart ID.'],
+    ['createCart', 'object', 'Maybe', 'Required if getCartId not provided. Should have a getCartItems function that returns a list with the cart items to purchase. The list must contain at least one cart item.'],
```

Comment:
```suggestion
    ['createCart', 'object', 'Maybe', 'Required if "getCartId" not provided. The object should define a single "getCartItems" property, which should be a function that returns a list of the cart items to purchase. The list must contain at least one cart item. Cart items must be provided as objects with, at minimum, an "sku" (string) and a "quantity" (number). See "Cart item object" section for more information.'],
```

with a dedicated section to specify what cart item objects should look like, e.g.,

<img width="789" height="912" alt="image" src="https://github.com/user-attachments/assets/5989f89a-1758-4cc1-b504-0718b07453b6" />

what do you think?